### DISCUSSION TITLE
VM expunge delay timers

### DISCUSSION THREAD

**Discussion by adietrich-ussignal** [apache/cloudstack#10113] [Q&A] [answered]
When destroying a VM, do subsequent attempts to destroy reset the expunge.delay timers? Seeing a user make subsequent attempts to destroy a VM, but passing the delay time for the initial destroy attempt has me thinking that is the case.

**Comment by DaanHoogland** [comment, marked as answer]
@adietrich-ussignal , I checked the flow and cannot find a reason this would happen. That doesn't mean It doesn't. If you can reproduce this scenario, please enter a [new issue](https://github.com/apache/cloudstack/issues/new) and describe this scenario in it.
